extensions/api/data-management/asset/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/asset/AssetApi.java

```diff
@@ -26,7 +27,7 @@
 @Tag(name = "Asset")
 public interface AssetApi {
 
-    void createAsset(AssetEntryDto assetEntryDto);
+    void createAsset(@Valid AssetEntryDto assetEntryDto);
```

Comment:
jakarta annotations should be defined only on implementations, leaving only swagger ones on the interfaces.

Reply:
unfortunately that won't work, as the validator will then raise an exception: 
```java
HV000151: A method overriding another method must not redefine the parameter constraint configuration, but method <XYZ> redefines the configuration of <XYZ>
```
so we're stuck with leaving it there, i'm afraid.